### PR TITLE
docs: add recommendation CLI documentation.

### PR DESCRIPTION
**Pull request by jrasell**
Provides documentation for CLI within #9272

### CODE REVIEW

**Comment by jrasell** [comment]
Vercel failure is the known problem when adding new pages. I don't have permission to redeploy.

File: website/pages/docs/commands/recommendation/apply.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## General Options
+
+@include 'general_options.mdx'
+
+## Apply Options
+
+- `-detach`: Return immediately instead of entering monitor mode. After applying
+  a recommendation, the evaluation ID will be printed to the screen, which can
+  be used to examine the evaluation using the eval-status command. If applying
```
Comment (cgbaker) [inline website/pages/docs/commands/recommendation/apply.mdx:34]:
```suggestion
  be used to examine the evaluation using the [`eval status`][/docs/commands/eval-status] command. If applying
```

**Review by cgbaker — APPROVED** [review]
thanks, mate 💯

File: website/pages/docs/commands/recommendation/apply.mdx (hunk 1 above)
Comment (cgbaker) [reply website/pages/docs/commands/recommendation/apply.mdx:34]: tiny change to wake up vercel, maybe

**Pull request merged**